packages/react-tabster/package.json

```diff
@@ -15,6 +15,7 @@
     "build": "just-scripts build",
     "clean": "just-scripts clean",
     "code-style": "just-scripts code-style",
+    "e2e": "e2e",
```

Comment:
These tests can finally be enabled on CI, because they are not a part of react-components storybook